.kitchen.docker.yml

```diff
@@ -1,21 +1,97 @@
 driver:
-  name: docker
+  name: dokken
+  privileged: true # because Docker and SystemD/Upstart
+  chef_image: chef/chef
+  chef_version: <%= ENV['CHEF_VERSION'] || '12.5.1' %>
```

Comment:
IIRC 12.5.1 is the 'earliest' the cookbook supports. I don't remember the functionality that was introduced in 12.5.1 but it was something...